web/modules/custom/rdf_serialization/rdf_serialization.services.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  rdf_serialization.encoder.jsonld:
+    class: Drupal\rdf_serialization\Encoder\RdfEncoder
+    tags:
+    - { name: encoder, format: jsonld }
+  rdf_serialization.encoder.rdfxml:
+    class: Drupal\rdf_serialization\Encoder\RdfEncoder
+    tags:
+      - { name: encoder, format: rdfxml }
+  rdf_serialization.encoder.ntriples:
+    class: Drupal\rdf_serialization\Encoder\RdfEncoder
+    tags:
+    - { name: encoder, format: ntriples }
+  rdf_serialization.encoder.turtle:
+    class: Drupal\rdf_serialization\Encoder\RdfEncoder
+    tags:
+    - { name: encoder, format: turtle }
+  rdf_serialization.encoder.n3:
+    class: Drupal\rdf_serialization\Encoder\RdfEncoder
+    tags:
+    - { name: encoder, format: n3 }
+  rdf_serialization.rdfsubscriber:
+    class: Drupal\rdf_serialization\EventSubscriber\RdfSubscriber
+    tags:
+      - { name: event_subscriber }
+  serializer.normalizer.entity.rdf:
+    class: Drupal\rdf_serialization\Normalizer\RdfEntityNormalizer
+    arguments: ['@rdf_export.serializer']
```

Comment:
For me this looks weird. Why depending on `rdf_export` module? I think this service should be moved in `rdf_serialization` and not the other way around. Then `rdf_export` would depend on `rdf_serialization`.